packages/core/src/odata-v4/payload-value-converter.ts

```diff
@@ -144,17 +143,15 @@ const deserializers: EdmTypeMapping = {
   'Edm.Date': edmDateToMoment,
   'Edm.DateTimeOffset': edmDateTimeOffsetToMoment,
   'Edm.Duration': edmDurationToMoment,
-  'Edm.TimeOfDay': edmTimeOfDayToTime,
-  'Edm.Enum': identity
+  'Edm.TimeOfDay': edmTimeOfDayToTime
```

Comment:
👍 Do we need a compatibility note? And while I think it is good to get rid of this, isn't this a breaking change?